modules/apps/portal-workflow/portal-workflow-kaleo-runtime-integration-impl/src/main/java/com/liferay/portal/workflow/kaleo/runtime/integration/internal/security/permission/WorkflowTaskPermissionImpl.java

```diff
@@ -78,19 +78,24 @@ public boolean contains(
 			return true;
 		}
 
-		int userNotificationEventsCount =
-			_userNotificationEventLocalService.getUserNotificationEventsCount(
-				permissionChecker.getUserId(), PortletKeys.MY_WORKFLOW_TASK,
-				HashMapBuilder.put(
-					"workflowInstanceId",
-					String.valueOf(workflowTask.getWorkflowInstanceId())
-				).put(
-					"workflowTaskId",
-					String.valueOf(workflowTask.getWorkflowTaskId())
-				).build());
+		boolean assignableUser;
```

Comment:
@ricardocousolr,

could you please init this variable as false and remove [this line](https://github.com/liferay-workflow/liferay-portal/pull/876/files#diff-64844be5f292a7fb6e7bad74f578f1230bbe5fa8202be568fb26db241b589307R94)?

Thank you,
Feliphe

Reply:
Hi @MarinhoFeliphe,

Thanks for your review. I'll add that change and update the PR with the additional commit.

Reply:
New commit with the requested change: [d4967c6](https://github.com/liferay-workflow/liferay-portal/pull/876/commits/d4967c6ef2cfccd4884a20536a730dcee5444233)

Reply:
Thank you!